ui/app/styles/_footer.scss

```
@@ -0,0 +1,33 @@
+
+@use 'pds/tokens/color';
+@use 'pds/tokens/scale';
+@use 'pds/tokens/font';
+@use 'pds/tokens/typography' as Typography;
+
+:root {
+  --footer-text: #{dehex(color.$ui-cool-gray-500)};
+}
+
+
+#footer {
+  color: rgb(var(--footer-text));
+  display: flex;
+  align-items: center;
+  font-size: 0.75rem;
+  margin: 0 auto;
+  width: 100vw;
+  max-width: 1600px;
+  padding: scale.$lg-3 scale.$lg-4 scale.$lg-5;
```

I'd reduce the top padding here as the #page element gives us quite a bit already.

```suggestion
  padding: scale.$sm-4 scale.$lg-4 scale.$lg-5;
```